README.md

```diff
@@ -2,7 +2,7 @@
 
 # compress-tools
 
-The library provide tools for handling compressed and archive files
+The library provides tools for handling compressed and archived files.
```

Comment:
The _provide_ is indeed a typo to be fixed, but I don't think _archived_ would be